internal/kafka/command_partition_reassignment_list.go

```diff
@@ -78,19 +78,19 @@ func (c *partitionCommand) reassignmentList(cmd *cobra.Command, args []string) e
 			return kafkarest.NewError(restClient.GetConfig().BasePath, err, resp)
 		}
 		if reassignmentGetResp.Kind != "" {
-			reassignmentListResp.Data = []kafkarestv3.ReassignmentData{reassignmentGetResp}
+			reassignments.Data = []kafkarestv3.ReassignmentData{reassignmentGetResp}
 		}
 	} else if topic != "" {
-		reassignmentListResp, resp, err = restClient.PartitionApi.ClustersClusterIdTopicsTopicNamePartitionsReassignmentGet(restContext, clusterId, topic)
+		reassignments, resp, err = restClient.PartitionApi.ClustersClusterIdTopicsTopicNamePartitionsReassignmentGet(restContext, clusterId, topic)
 	} else {
-		reassignmentListResp, resp, err = restClient.PartitionApi.ClustersClusterIdTopicsPartitionsReassignmentGet(restContext, clusterId)
+		reassignments, resp, err = restClient.PartitionApi.ClustersClusterIdTopicsPartitionsReassignmentGet(restContext, clusterId)
 	}
 	if err != nil {
 		return kafkarest.NewError(restClient.GetConfig().BasePath, err, resp)
 	}
 
 	list := output.NewList(cmd)
-	for _, reassignment := range reassignmentListResp.Data {
+	for _, reassignment := range reassignments.Data {
```

Comment:
Looks like we'll need to write wrapper functions for this sdk if we want to remove these later, right?

Reply:
Yep!